gnocchi/rest/api.py

```diff
@@ -157,7 +159,10 @@ def deserialize(expected_content_types=None):
     try:
         params = json.load(pecan.request.body_file)
     except Exception as e:
-        abort(400, "Unable to decode body: " + six.text_type(e))
+        details = "Unable to decode body: %s \n" \
```

Comment:
do you actually want that new line in the message? is there a common practice on how to escape them because i find multi-line logs to be unusable as when you filter, most systems end up returning only the matching line and you have an incomplete log.

Reply:
I also did not like that multi-line logging. Maybe enclosing the values in "[]" in the log string can help. Such as `Unable to decode body: [%s], request body [%s]`.

Reply:
that could work, i'd think log filtering tools should be able to parse that.

we did start transitioning our errors to be json (search on jsonify). this was mainly for api responses but it could also work for logs as well i believe? maybe @jd has an opinion on it but as long as there's no newline, it works for me :)